rasa/utils/tensorflow/constants.py

```diff
@@ -2,6 +2,7 @@
 
 LABEL = "label"
 IDS = "ids"
+LABEL_PAD_ID = -1
```

Comment:
How does 
```python
# LABEL_PAD_ID is used to pad multi-label training examples.
# It should be <0 to avoid index out of bounds errors by tf.one_hot. 
LABEL_PAD_ID = -1
```

sound?